### ISSUE TITLE
[torch-neuron] Mask-RCNN, Faster-RCNN Object detection models inference support on Inf1

### ISSUE THREAD

**Issue by AWSGH** [aws-neuron/aws-neuron-sdk#153]

**Comment by ds-mike** [comment]
To be clear, should I expect this enhancement will allow for Mask/Faster-RCNN TF object detection models trained via [this API](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md) to be compiled for neuron/AWS inferentia machines?

I'm attempting to compile one now and hitting a variety of errors when using either `tfn.trace`, `tfn.saved_model.trace`, or `tfn.saved_model.compile`. It's not clear to me these models can be compiled at the moment, but it sounds like this enhancement will allow for this.

Please let me know if I understand this right, appreciate your input.

**Comment by awsrjh** [comment]
Hi - this is still an open item on the roadmap, so RCNN support is still limited.

**Comment by aws-maens** [comment]
Closing this issues, RNNs are not a good fit for Inferentia - see https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/model-architecture-fit.html

**Comment by aws-maens** [comment]
Released in Neuron 2.5.0
